app/src/main/java/io/homeassistant/companion/android/widgets/WidgetDynamicFieldAdapter.kt

```diff
@@ -118,4 +145,54 @@ class WidgetDynamicFieldAdapter(
             override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {}
         })
     }
+
+    private fun String.toJsonType(): Any? {
+        // Parse the string to one of the following
+        // valid base JSON types:
+        // array, number, boolean, string
+
+        // Check if the text is an array
+        if (this.contains(",")) {
+            val jsonArray = ArrayList<Any>()
+
+            this.split(",").forEach { subString ->
```

Comment:
What happens when the language has `1.1` = `1,1`?

Reply:
After thinking about this more I think we can expect that they use `.` because that's what the JSON format supports.